src/index.js

```diff
@@ -30,4 +29,82 @@ const csvDownload = (data, name, delimiter) => {
   document.body.removeChild(link)
 }
 
-export default csvDownload
+/**
+ * Provides support for digesting and exporting JSON to CSV
+ * 
+ * @param {JSON} __data 
+ * @param {String} __name 
+ * @param {Char} __delimiter 
+ * @returns Object
+ */
+export const jsonToCsv = (__data, __name, __delimiter) => {
```

Comment:
nit: I'm not a fan of underscore prefixes for regular args

Reply:
It would be advantageous to make all the new function args named parameters for ease of use. (E.g. no name override, but a delimiter.)